libshaderc/include/shaderc/shaderc.h

```diff
@@ -122,7 +122,8 @@ typedef enum {
 
 // Optimization level.
 typedef enum {
-  shaderc_optimization_level_zero,  // no optimization
+  shaderc_optimization_level_zero,         // no optimization
+  shaderc_optimization_level_performance,  // optimize towards performance
```

Comment:
Don't insert a value in between two existing values.  That breaks compatibility with anybody who has code that is compiled against the old header. Their code might ask for size optimization and now they would end up getting performance.

Reply:
Urgh. Yes. I don't know what I was doing...